diff --git a/jira/client.py b/jira/client.py
index 0af6c29..2309b37 100644
--- a/jira/client.py
+++ b/jira/client.py
@@ -518,7 +518,11 @@ class JIRA:
         if len(context_path) > 0:
             self._options["context_path"] = context_path
 
-        self._try_magic()
+        # This is patched out for Fedora because python-jira is expecting a
+        # different magic module than the one we ship.  See:
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1025368#c11
+        #self._try_magic()
+        self._magic = None
 
         assert isinstance(self._options["headers"], dict)  # for mypy benefit
         self._session: ResilientSession  # for mypy benefit
