From baa04780f473ceb1dcd2af43dd6ee61fd56b3a2b Mon Sep 17 00:00:00 2001
From: Ralph Bean <rbean@redhat.com>
Date: Tue, 10 May 2016 13:00:32 -0400
Subject: [PATCH] no mime detection

---
 jira/client.py | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/jira/client.py b/jira/client.py
index 6dc54d5..110f453 100644
--- a/jira/client.py
+++ b/jira/client.py
@@ -234,7 +234,11 @@ class JIRA(object):
         if len(context_path) > 0:
             self._options['context_path'] = context_path
 
-        self._try_magic()
+        # This is patched out for Fedora because python-jira is expecting a
+        # different magic module than the one we ship.  See:
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1025368#c11
+        #self._try_magic()
+        self._magic = None
 
         if oauth:
             self._create_oauth_session(oauth)
-- 
2.5.5

